### PR TITLE
Remove cli after move to orion repo

### LINKED ISSUE #112: Migrate cli artifacts to its own git repo
### Description
*As a developer, I'd like to pull the cli code artifacts from the same repo as the test harness and documentation site. 

### Acceptance Criteria
Reference: [Done-Done Checklist] (https://github.com/Microsoft/code-with-engineering-playbook/blob/master/Engineering/BestPractices/DoneDone.md)
- [ ] CLI code is migrated to the same repo as the documentation site. 

1. move `cli` folder to new repo
2. commit to new repo
3. remove `cli` from cobalt repo and pr

### PR DESCRIPTION
**Pull request by iphilpot**
## All Submissions:
-------------------------------------
* [YES] Have you followed the guidelines in our Contributing [document](./CONTRIBUTING.md)?
* [YES] Have you added an explanation of what your changes do and why you'd like us to include them?
* [YES] I have updated the documentation accordingly.
* [NA] I have added tests to cover my changes.
* [NA] All new and existing tests passed.
* [NA] My code follows the code style of this project.
* [NA] I ran lint checks locally prior to submission.
* [YES] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

## What is the current behavior?
-------------------------------------
Remove CLI folder and code after migration to orion repo

Issue Number: 112

## What is the new behavior?
-------------------------------------
Move CLI to new repo removes from current

## Does this introduce a breaking change?
-------------------------------------
- [NO]

## Any relevant logs, error output, etc?
-------------------------------------
 - na

## Other information
-------------------------------------
[orion](https://github.com/Microsoft/orion)

closes #112

### CODE REVIEW

File: infra/modules/providers/azure/cosmosdb/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Module Azure Cosmos DB
```
Comment (erikschlegel) [inline infra/modules/providers/azure/cosmosdb/README.md]: can you confirm that you rebased from master?

Comment (iphilpot) [reply infra/modules/providers/azure/cosmosdb/README.md]: Yes, rebased from master this morning.

Comment (erikschlegel) [reply infra/modules/providers/azure/cosmosdb/README.md]: Why are the cosmos module changes marked as additions in this commit?

Comment (iphilpot) [reply infra/modules/providers/azure/cosmosdb/README.md]: I think we missed the window and need a rebase again... oy

Comment (erikschlegel) [reply infra/modules/providers/azure/cosmosdb/README.md]: Apologies for the delay @iphilpot . Promise I'll review as soon as you post a new commit.

**Pull request merged**